src/kirby/components/page/page.component.ts

```diff
@@ -109,6 +110,7 @@ export class PageComponent implements OnInit, OnDestroy, AfterViewInit {
   @Output() enter = new EventEmitter<void>();
   @Output() leave = new EventEmitter<void>();
 
+  @ViewChild('content', { static: true }) private content: IonContent;
```

Comment:
Consider just selecting on Type as there is only 1 `ion-content`
```suggestion
  @ViewChild(IonContent, { static: true }) private content: IonContent;
```